src/Helpers/Debugger.php

```diff
@@ -1,20 +1,17 @@
-<?php
+<?php declare(strict_types = 1);
 
 namespace Contributte\Scheduler\Helpers;
 
-use Exception;
+use Throwable;
+use Tracy\Debugger as TracyDebugger;
 
 class Debugger
 {
 
-	/**
-	 * @param Exception $e
-	 * @return void
-	 */
-	public static function log(Exception $e)
+	public static function log(Throwable $e): void
 	{
-		if (class_exists('\Tracy\Debugger')) {
-			\Tracy\Debugger::log($e);
+		if (class_exists(TracyDebugger::class)) {
```

Comment:
Does it really works without the tracy/tracy package? I'm not sure. Have you tested it?

Reply:
Yes, autoloading is not triggered by use statements, safe to use everywhere :)

Reply:
http://sandbox.onlinephpfunctions.com/code/1d3d9fa8a8199e9bdabb60570cb73e87280bf412

Reply:
Cool. thanks.